samples/CMakeLists.txt

```
@@ -1,9 +1,12 @@
-# Copyright (c) 2024 CATIE
+# Copyright (c) 2024, CATIE
 # SPDX-License-Identifier: Apache-2.0
 
 cmake_minimum_required(VERSION 3.20.0)
+
+set(SHIELD zest_display_lcd)
+
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(ili9163c)
 
-FILE(GLOB app_sources src/main.c)
+FILE(GLOB app_sources src/*.c)
```

There is only one `.c` file 
```suggestion
FILE(GLOB app_sources src/main.c)
```

---

Resolved in [9d3e388](https://github.com/catie-aq/zephyr_ilitek-ili9163c/pull/3/commits/9d3e388f593571c4797e21606e0134fa4e9482c9).